src/OOVPADatabase/D3D8LTCG/4039.inl

```diff
@@ -85,6 +85,33 @@ OOVPA_SIG_MATCH(
     // Do not use any offsets after 0x1B
 );
 
+// test case: NBA2K2
```

Comment:
nit: Move below the title symbol comment, not above. testcase is one word for internal documentation.